source/reference/options.txt

```diff
@@ -20,8 +20,12 @@ General Options
 
 .. option:: --build-info
 
-   Returns a JSON-formatted document with information about the
-   :binary:`~bin.mongosh` build.
+   Returns a JSON-formatted document with build information and driver 
+   dependencies about your :binary:`~bin.mongosh` binary.
```

Comment:
[suggestion] The wording here sounds a bit off with "driver dependencies about..." Maybe we can make it consistent with the other description above?

Something along the lines of: "Returns the a JSON-formatted document with information about
your ``mongosh`` build and driver dependencies."